### ISSUE TITLE
autocomplete working ?

### ISSUE THREAD

**Issue by angeenes** [Brakebein/ngx-tagify#5]
Hi, does the autocomplete is supposed to work in the input field as you type ? Like in here for example : [tagify](https://codepen.io/vsync/pen/VKBbdv)

Can't make it work with this tagify Angular wrapper.

Example  in this image of what it is supposed to do : 

![74661211-cd083480-5197-11ea-8b06-14fc0d15a96d](https://user-images.githubusercontent.com/6044086/118290873-3419b380-b4d7-11eb-8fd9-6ac2f1c79bbc.png)

**Comment by Brakebein** [comment]
Do you mean that the input field should show the completed part of a suggestion in gray color? I can't see any difference in behavior between your codepen example and the [ngx-tagify demo](https://brakebein.github.io/ngx-tagify/) (see "Another tagify" example). If you start typing `Hello` or `World` (at least 3 characters), it will show the rest of the word in gray. If your start with lowercase (`hello` or `world`), the gray part will only be shown if you hover or select a suggestion. So, it's the same behavior as with the original tagify library.

Or do you mean something else?

**Comment by angeenes** [comment]
> Do you mean that the input field should show the completed part of a suggestion in gray color? I

Yes

I actually resolved it. It was on my side.

For those who have the same "issue" :
I changed the Sass variable $tag-text-color in white ( #fff ) which resulted in being the same color as the input background-color so I could not see it even if in the dom html I could see the "data-suggest" working... Which confused me even more to find the problem that i caused.

this is actually more a Tagify related question than from your wrapper.

Thanks for the quick reply and to have made this usefull tagify wrapper and keep up the good work. :-)
